Content.Client/Lobby/UI/LobbyCharacterPreviewPanel.cs

```diff
@@ -141,29 +141,26 @@ public void UpdateUI()
         public static void GiveDummyJobClothes(EntityUid dummy, HumanoidCharacterProfile profile)
         {
             var protoMan = IoCManager.Resolve<IPrototypeManager>();
-
             var entMan = IoCManager.Resolve<IEntityManager>();
-            var inventory = entMan.GetComponent<ClientInventoryComponent>(dummy);
+            var invSystem = EntitySystem.Get<ClientInventorySystem>();
 
             var highPriorityJob = profile.JobPriorities.FirstOrDefault(p => p.Value == JobPriority.High).Key;
 
             // ReSharper disable once ConstantNullCoalescingCondition
             var job = protoMan.Index<JobPrototype>(highPriorityJob ?? SharedGameTicker.FallbackOverflowJob);
 
-            inventory.ClearAllSlotVisuals();
-
-            if (job.StartingGear != null)
+            if (job.StartingGear != null && invSystem.TryGetSlots(dummy, out var slots))
             {
                 var gear = protoMan.Index<StartingGearPrototype>(job.StartingGear);
 
-                foreach (var slot in AllSlots)
+                foreach (var slot in slots)
                 {
-                    var itemType = gear.GetGear(slot, profile);
+                    var itemType = gear.GetGear(slot.Name, profile);
                     if (itemType != string.Empty)
                     {
+                        invSystem.TryUnequip(dummy, slot.Name, true, true);
```

Comment:
Well it works now... but I can't help but feel that something aint right here
![something aint right here](https://user-images.githubusercontent.com/60421075/147023821-48754e5b-560b-4952-a53b-67c4a0beaed4.png)

Reply:
(should probably call `TryUnequip` even if string is empty)

Reply:
lmfao

Reply:
also added having the item deleted when dropped